src/components/AuthenticatedNavBar.tsx

```diff
@@ -1,13 +1,15 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
 import { SignOutButton } from "../api";
+import Navbar from "react-bootstrap/Navbar";
+import Container from "react-bootstrap/Container";
 
 import "./AuthenticatedNavBar.css";
 
 export function AuthenticatedNavBar() {
 	return (
-		<nav className="Nav">
-			<div className="Nav-container">
+		<Navbar className="Nav">
```

Comment:
Writing this here, so I hopefully don't forget: There should be a Navlink Container Bootstrap component. A good "tomorrow problem"!

Reply:
Great point 😅 I was nervous to mess with the navbar too much it was late and I couldn't wrap my head aorund it pass the Navbar and container!